### PR TITLE
Add function withdraw item, Fix: deposit coin_value must be an integer

### PR DESCRIPTION
**Pull request by tuanngodoan**
- Add function withdraw Non-auction item
- Fix error: deposit coin_value must be an integer

### CODE REVIEW

**Comment by Art-em1s** [comment]
Hi, thanks for your PR.

I'll try and check this and get back to you by the end of the week.

File: withdrawals.py [hunk 1]
```diff
@@ -33,6 +33,22 @@ def bid(self, item_id: int, amount: int):
...
+        status = response.status_code
+        response = response.json()
+
+        if status == 200:
+            return True
+        else:
+            if status == 401:
+                raise InvalidApiKey()
+            elif status == 429:
+                raise ExceedsRatelimit(f"Withdrawal:Non-Bid: {response['message']}")
```
Comment (Art-em1s) [inline withdrawals.py]: Can "Non-bid" be changed to "Buy" just to be less confusing

File: withdrawals.py [hunk 2]
```diff
@@ -33,6 +33,22 @@ def bid(self, item_id: int, amount: int):
...
+            else:
+                raise RequestError(f"Withdrawal:Non-Bid: {response['message']}")
```
Comment (Art-em1s) [inline withdrawals.py]: Can "Non-bid" be changed to "Buy" just to be less confusing

**Comment by tuanngodoan** [comment]
@Art-em1s I resolved that comment. Thanks

**Pull request merged**